test/tests/amqp/test_amqp_poller_alert.py

```diff
@@ -0,0 +1,364 @@
+                "Fail to apply obm setting!")
+        # bmcip=self._apply_obmsetting_to_node(nodeid)
+        test_api_utils.run_ipmi_command_to_node(nodeid, "sel clear")
+        # Reboot the node to begin rediscover.
+        pollerid = self._get_ipmi_poller_by_node(nodeid, "selEntries")
+        logs.debug('launch AMQP thread for sel alert')
+        sel_worker = AmqpWorker(
+            exchange_name="on.events",
+            topic_routing_key="polleralert.sel.#." + nodeid,
+            external_callback=self.amqp_callback,
+            timeout=200)
+        sel_worker.setDaemon(True)
+        sel_worker.start()
+        # Send out sel iERR injection.
+        command = "raw 0x0a 0x44 0x01 0x00 0x02 0xab 0xcd 0xef 0x00 0x01 0x00 0x04 0x07 0x02 0xef 0x00 0x00 0x00"
+        test_api_utils.run_ipmi_command_to_node(nodeid, command)
+        if skupack_intalled:
+            self._wait_amqp_message(200)
+            workflow_amqp = amqp_queue.get()
+            self._process_message(
+                "sel.updated",
+                pollerid,
+                nodeid,
+                "polleralert",
+                "critical",
+                workflow_amqp)
+            sel_worker.dispose()
+
+    def test_chassis_power_updated(self):
+        node_collection = test_api_utils.get_node_list_by_type("compute")
+        nodeid = ""
+        skupack_intalled = self.check_skupack()
+        nodeid = node_collection[
+            random.randint(
+                0, len(node_collection) - 1)]
+        logs.debug_2('Checking OBM setting...')
+        node_obm = fit_common.rackhdapi(
+            '/api/2.0/nodes/' + nodeid)['json']['obms']
+        if node_obm == []:
+            self.assertTrue(
+                self._apply_obmsetting_to_node(nodeid),
+                "Fail to apply obm setting!")
+        pollerid = self._get_ipmi_poller_by_node(nodeid, "chassis")
+        logs.debug('launch AMQP thread for chassis state monitor')
+        chassis_worker = AmqpWorker(
+            exchange_name="on.events",
+            topic_routing_key="polleralert.chassispower.updated.#",
+            external_callback=self.amqp_callback,
+            timeout=200)
+        chassis_worker.setDaemon(True)
+        chassis_worker.start()
+        ssh_output = test_api_utils.run_ipmi_command_to_node(nodeid, "chassis power status")
+        # If in initial state the chassis power is on, then the test will do power off->power on
+        # Otherwise it will do power on -> power off
+        if "Chassis Power is on" in ssh_output['stdout']:
+            command1 = "chassis power off"
+            command2 = "chassis power on"
+        else:
+            command1 = "chassis power on"
+            command2 = "chassis power off"
+        logs.debug('Power off/on node to change chassis power state')
+        test_api_utils.run_ipmi_command_to_node(nodeid, command1)
+        logs.debug('Wait chassis power update')
+        if skupack_intalled:

```

Comment:
Good catch! Just checked the sku pack only apply to sel alert. No impact on power status.